tests/test_miner.py

```diff
@@ -0,0 +1,144 @@
+"""Test the miner module."""
+import pytest
+import pandas as pd
+
+from orangecontrib.argument.miner.miner import (
+    select_by_topic,
+    get_edges,
+    get_edge_weights,
+    get_edge_table,
+    get_node_labels,
+    get_node_table,
+)
+from .conftest import dummy_argument_selection, dummy_edge_data, df_arguments_processed
+
+
+@pytest.mark.parametrize(
+    "dummy_data, dummy_topic, expected_result",
+    [
+        (
+            pd.DataFrame({"topics": [(0), (1, 2)]}),
+            0,
+            pd.DataFrame({"topics": [(0)], "argument_id": [0]}),
+        ),
+        (
+            pd.DataFrame({"topics": [(), (1, 2)]}),
+            0,
+            pd.DataFrame({"topics": [], "argument_id": []}),
+        ),
+        (
+            pd.DataFrame({"topics": ["(0)", "(1, 2)"]}),
+            0,
+            pd.DataFrame({"topics": ["(0)"], "argument_id": [0]}),
+        ),
+    ],
+)
```

Comment:
I would consider moving this to the conftest as well, and importing it with other dummy data